server/models.py

```diff
@@ -24,5 +26,20 @@ def __init__(self, *data, **kwargs):
         for key in kwargs:
             setattr(self, key, kwargs[key])
 
-# class Users(dbs.Model):
-#     pass
+
+class User(dbs.Model):
+    id = dbs.Column(dbs.Integer, primary_key=True)
```

Comment:
Add a constructor

Reply:
I am actually ignorant on why we are defining our constructors for our tables. Why are we doing that? For `Qualifying_Answers`, I think you said it was because of how we were receiving data from the front end. Are you suggesting we would be doing the same for User logins?

Reply:
The difference lies in how the object is created. 

`usr = User(username=response.json.username, email=response.json.email, pw_hash=hashFunc(???), submissions=Submission())` 

versus

```
some_dict = {
    'username': response.json.username
    ...
}

usr = User(some_dict)
```

Adding a simple constructor seems like it would be a little cleaner.